File: SPACE TRIANGLE PATTERN/code2.c
#include <stdio.h>

void main (){

	int row ;
	printf("Enter no. of rows : ");
	scanf("%d",&row);

	for(int i = 0 ; i < row ; i++){

		for(int a = 0 ; a < row - i ; a++){

			printf(" \t");
		}

		for(int j = 0; j <= i ; j++){

			printf("%c\t",(65 + i + j ));

		}

		printf("\n\n");

	}


}

File: JAVA DAILYCODES ASSGNPRATICE QUIZE DSA/Basic is else assignments/8/code2.java
import java.util.Scanner;

class NestedDemo{

	public static void main(String[] args){

		Scanner s = new Scanner(System.in);
		System.out.println("enter no. of rows");
		int row = s.nextInt();
		int temp =1 ;

		for(int i = 0; i < row ; i++){
			for(int j = 1;j <= row;j++){

				System.out.print((temp*temp)+"	");
				temp++;
			}
			System.out.println("\n");
		}
	}
}
